<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDelayDialog</class>
 <widget class="QDialog" name="SetDelayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BtnAccept">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDelay">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IntroLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>12</pointsize>
     <fontweight>Medium</fontweight>
    </font>
   </property>
   <property name="text">
    <string>请输入延迟时间（单位为ms）</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
